src/proxy/app/authorize.py

```diff
@@ -75,7 +75,7 @@ async def __is_user_authorized(self, api_key: UUID, deployment_name: str) -> Aut
                 detail="Authentication failed.",
             ) from exception
 
-    # @lru_cache_with_expiry(maxsize=128, ttl=180)
+    @lru_cache_with_expiry(maxsize=128, ttl=10)
```

Comment:
It might be helpful to comment as to the reasons for the TTL, especially as I see you use a different TTL in other parts of the code.